requirements-aarch64.txt

```diff
@@ -210,8 +208,6 @@ langsmith==0.1.75
     #   langchain-core
 launchdarkly-server-sdk==8.3.0
     # via -r requirements.in
-levenshtein==0.25.1
-    # via ansible-risk-insight
```

Comment:
I assume this was updated in a previous PR and there wasn't a corresponding aarch64 requirements.in update

Reply:
Yes, this is a consequence of b83fd4c23d8f142ac7f828cc5a476b626b5d61fd where only the `x86_64` was adjusted. It's an harmless change.

Reply:
Follow up PR https://github.com/ansible/ansible-ai-connect-service/pull/1139